tsl/test/expected/chunk_utils_internal.out

```diff
@@ -119,12 +165,40 @@ FROM _timescaledb_catalog.chunk WHERE table_name = :'CHUNK_NAME';
  _hyper_1_1_chunk |      4
 (1 row)
 
+-- Check trigger
```

Comment:
can drop this check since we verified earlier

Reply:
The check is removed